### DISCUSSION TITLE
UM Physics Fork

### DISCUSSION THREAD

**Discussion by yaswant** [MetOffice/simulation-systems#285] [Announcements]
## UM Physics Fork

You will all be aware that the conversation on whether to fork the UM Physics has been going on for some years. The time is finally deemed to be right, both with the improvements to the processes around LFRic Apps, and with the need to develop GPU benchmarks for the LFRic Atmosphere model. Therefore, we are aiming to create a fork of the physics as the last change committed at the February/March release. 
 
What this means in practice is that a copy of all the UM code used by LFRic Apps will be placed within the LFRic Apps repository. Changes to this code can either be carried out just in the LFRic Apps copy, or duplicated in both the UM and LFRic Apps as needed. Improvements should not be made to just the UM. Over time this means the code bases will diverge. 

JULES, UKCA, SOCRATES and CASIM code will remain untouched in their separate repositories and will continue to interface with both the UM and LFRic Apps.  More details will follow as we go.

Please use this space or 📧[get in touch](mailto:UMSysTeam@metoffice.gov.uk) with questions and concerns. 

## Draft Schedule (_stc_)

![image](https://github.com/user-attachments/assets/1e70dd07-727a-431c-b2f9-435a69fd6843)

**Comment by jennyhickson** [comment]
This work was completed as part of LFRic Apps vn2.1 last week. Further details on the implementation can be found [here](https://code.metoffice.gov.uk/trac/lfric_apps/wiki/UMPhysicsFork).

Thanks
